dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/translator/visitor/generator/ImportGenerator.java

```diff
@@ -38,6 +39,33 @@ public void startGeneration(int section, JavaCodeWriter writer)
 					}
 				}
 			}
+		} else if (section == CodeGenerationPhase.STATIC_SECTION && PagesVersionHandler.isPages31OrHigherLoaded()) {
+			NamedNodeMap attributes = element.getAttributes();
+			if (attributes != null) {
+				for (int i = 0; i < attributes.getLength(); i++) {
+					Node attribute = attributes.item(i);
+					String directiveName = attribute.getNodeName();
+					String directiveValue = attribute.getNodeValue();
+					if (directiveName.equals("import")) {
+						StringTokenizer tokenizer = new StringTokenizer(directiveValue, ",");
+						writeDebugStartBegin(writer);
+						while (tokenizer.hasMoreTokens()) {
+							String singleImport = ((String) tokenizer.nextToken()).trim();
+							if(singleImport.endsWith(".*")){
+								writer.println("importPackageList.add(\"" + singleImport.replace(".*", "") + "\");");
+							} else {
+								writer.println("importClassList.add(\"" + singleImport + "\");");
+							}
```

Comment:
Only idea I could come up with was to differentiate packages / class was with .*

Reply:
Oh, let add a test for this import.

Reply:
Spec says ...  `or of a package name followed by the .*
string, denoting all the public types declared in that package. `

I think this is ok

Reply:
@volosied did you add a test?

Reply:
Looks good: <%@ page import = "io.openliberty.pages31.fat.misc.Coffee,io.openliberty.pages31.fat.misc.Cafe,io.openliberty.pages31.fat.misc.other.*" %>